Core/DISET/private/BaseClient.py

```diff
@@ -47,6 +47,9 @@ def __init__( self, serviceName, **kwargs ):
     self._destinationSrv = serviceName
     self._serviceName = serviceName
     self.kwargs = kwargs
+    self.__useCertificates = None
+    # The CS useServerCertificate option can be overridden by explicit argument
+    self.forceUseCertificates = self.kwargs.get( self.KW_USE_CERTIFICATES )
```

Comment:
Maybe this one also deserves to be private. Sorry I was not clear in my previous comment